README.md

```diff
@@ -172,39 +177,129 @@ Note that you should have [Nvidia Docker Toolkit](https://github.com/NVIDIA/nvid
 # If you use the MobileNet model run this instead:
 # ./download_x86_model.sh
 
-# 1) Build Docker image (This step is optional, you can skip it if you want to pull the container from neuralet dockerhub)
+# 1) Build Docker image
 docker build -f x86-gpu.Dockerfile -t "neuralet/smart-social-distancing:latest-x86_64_gpu" .
 
 # 2) Run Docker container:
 Notice: you must have Docker >= 19.03 to run the container with `--gpus` flag.
 docker run -it --gpus all -p HOST_PORT:8000 -v "$PWD":/repo -e TZ=`./timezone.sh` neuralet/smart-social-distancing:latest-x86_64_gpu
 ```
 
-##### Run on x86 with GPU using TensorRT optimization
+###### Run on x86 with GPU using TensorRT optimization
 
 Note that you should have [Nvidia Docker Toolkit](https://github.com/NVIDIA/nvidia-docker) to run the app with GPU support
 ```bash
 
 
-# 1) Build Docker image (This step is optional, you can skip it if you want to pull the container from neuralet dockerhub)
+# 1) Build Docker image
 docker build -f x86-gpu-tensorrt-openpifpaf.Dockerfile -t "neuralet/smart-social-distancing:latest-x86_64_gpu_tensorrt" .
 
 # 2) Run Docker container:
 # Notice: you must have Docker >= 19.03 to run the container with `--gpus` flag.
 docker run -it --gpus all -p HOST_PORT:8000 -v "$PWD":/repo -e TZ=`./timezone.sh` neuralet/smart-social-distancing:latest-x86_64_gpu_tensorrt
 ```
 
-##### Run on x86 using OpenVino
+###### Run on x86 using OpenVino
 ```bash
 # download model first
 ./download_openvino_model.sh
 
-# 1) Build Docker image (This step is optional, you can skip it if you want to pull the container from neuralet dockerhub)
+# 1) Build Docker image
 docker build -f x86-openvino.Dockerfile -t "neuralet/smart-social-distancing:latest-x86_64_openvino" .
 
 # 2) Run Docker container:
 docker run -it -p HOST_PORT:8000 -v "$PWD":/repo  -e TZ=`./timezone.sh` neuralet/smart-social-distancing:latest-x86_64_openvino
 ```
+
+##### Running the processor from neuralet Docker Hub repository
+
+Before running any of the images available in the Docker repository, you need to follow these steps to have your device ready.
+  1. Create a `data` folder.
+  2. Copy the `config` file (available in this repository) corresponding to your device.
+  3. Copy the bash script(s) (available in this repository) required to download the model(s) your device requires.
```

Comment:
They also need the timezone.sh script, right? I would mention that here

Reply:
As the timezone.sh script is an optional parameter I didn't include it in the default commands.
I  explained how to use it with docker hub in his section. However, I can include it.

Reply:
I agree it would be best to do so (and remark its optionality here), this part is where I expect one would be interested of knowing and looking for the script If they want it (instead of after finishing the setup and attempting a run)

Reply:
I included the `timezone.sh` script.